netplugin/netd.go

```diff
@@ -14,248 +14,295 @@ limitations under the License.
 package main
 
 import (
-	"flag"
 	"fmt"
 	"log/syslog"
 	"net/url"
 	"os"
-	"os/user"
+	"sort"
 	"strings"
 	"time"
 
 	"github.com/contiv/netplugin/core"
+	"github.com/contiv/netplugin/netmaster/master"
 	"github.com/contiv/netplugin/netplugin/agent"
 	"github.com/contiv/netplugin/netplugin/cluster"
 	"github.com/contiv/netplugin/netplugin/plugin"
+	"github.com/contiv/netplugin/utils"
 	"github.com/contiv/netplugin/version"
 
-	log "github.com/Sirupsen/logrus"
+	"github.com/Sirupsen/logrus"
 	"github.com/Sirupsen/logrus/hooks/syslog"
+	"github.com/urfave/cli"
 )
 
-// StringSlice is a flag option implementation
-type StringSlice []string
-
-func (s *StringSlice) String() string {
-	return fmt.Sprintf("%s", *s)
-}
-
-// Get returns the slice of strings set by this flag
-func (s *StringSlice) Get() interface{} {
-	return *s
-}
-
-// Set function for StringSlice sets the appropriate field for flag handling
-func (s *StringSlice) Set(value string) error {
-	optVal := strings.Split(value, ",")
-	for _, option := range optVal {
-		if !strings.Contains(s.String(), option) {
-			*s = append(*s, option)
-		}
-	}
-	return nil
-}
-
-// Value returns the slice of strings set by this flag
-func (s *StringSlice) Value() []string {
-	return *s
-}
-
-// a daemon based on etcd client's Watch interface to trigger plugin's
-// network provisioning interfaces
-
-type cliOpts struct {
-	hostLabel    string
-	pluginMode   string // plugin could be docker | kubernetes | swarm-mode
-	cfgFile      string
```

Comment:
looks like you dropped this capability

Reply:
it doesn't seem to be used from what i see

Reply:
it's something that has to appear in release notes and commit message though, in case someone else was . . . not sure how though, maybe legacy

Reply:
done in commit msg